### ISSUE TITLE
Precice adapator 

### ISSUE THREAD

**Issue by tapegoji** [nemocrys/pyelmer#37]
Hi,

This is not an issue, only a suggestion to include precice adaptor into pyelmer. Do you have any roadmap for that?

**Comment by arvedes** [comment]
This is not planned, as this application is too specific and would mess up the code of pyelmer too much. Usually, the adapters for preCICE are maintained in their own repository. You could still use pyelmer to configure the required solver in Elmer for the adapter, though :)

Unfortunately, the Elmer adapter has not been further developed for quite some time, and its functionality is still very limited; I hope it is nevertheless useful for you.

**Comment by tapegoji** [comment]
Thank you for the reply. Yes, the adapter is old and now with v3 is quite useless.  I know you did some work on the adapter. did you do any more work on that?
Also, i'd like to connect with you and have a chat if you have some time. How can I reach you?

**Comment by arvedes** [comment]
No, I don't plan to work any further on this. We decided to apply a more straightforward coupling approach for Elmer and OpenFOAM by using the steady-state/time-averaged flow field as an input for Elmer and [elmerToFoamBC](https://github.com/nemocrys/elmerToFoamBC) to set the steady-state heat fluxes/temperatures on the boundaries in OpenFOAM.

However, it would be very nice to have a working preCICE adapter at hand, especially for fully unsteady cases. Did you notice [EOF-Library](https://github.com/jvencels/EOF-Library)? It's a very nice tool for the volume coupling of Elmer and OpenFOAM.

Please feel free to write me an e-mail at arved.wintzer@ikz-berlin.de.
